main.js

```diff
@@ -59,7 +76,57 @@ function getCurrentGutter() {
 	return getGridProperties().gutter;
 }
 
+/**
+ * This sets MediaQueryListeners on all the o-grid breakpoints
+ * and fires a `o-grid.layoutChange` event on layout change.
+ */
+function addBreakpointListeners() {
+	try {
+		// Create a map containing all breakpoints exposed via html:before
+		const breakpoints = new Map(Object.entries(getGridBreakpoints().layouts));
```

Comment:
In the [`browserFeatures` section in origami.json](https://github.com/Financial-Times/o-grid/blob/master/origami.json#L10) we will need to add `Map` as a required feature.